configs/pvrcnn/pvrcnn_8xb2-80e_kitti-3d-3class.py

```diff
@@ -0,0 +1,353 @@
+_base_ = [
+    '../_base_/datasets/kitti-3d-3class.py',
+    '../_base_/schedules/cyclic-40e.py', '../_base_/default_runtime.py'
+]
+
+voxel_size = [0.05, 0.05, 0.1]
+point_cloud_range = [0, -40, -3, 70.4, 40, 1]
```

Comment:
The `configs/pvrcnn/pvrcnn_8xb2-80e_kitti-3d-3class.py` should not be different in the diff view. Please pull the latest dev-1.x branch since the dev-1.x branch has been rebased.